packages/cw2-supported-interface/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "cw2-supported-interface"
```

Comment:
I think we can name it `cw22` or something along those lines, to follow the current convention.